pulsar-client-admin/src/main/resources/mltool/MLDataFormats_pb2.py

```diff
@@ -0,0 +1,342 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Make sure to that the maven license plugin will ignore this, otherwise maven install will fail